app/src/main/java/com/jerboa/ui/components/post/PostListing.kt

```diff
@@ -1230,29 +1236,45 @@ private fun ThumbnailTile(
                 }
             }
 
-        postView.post.thumbnail_url?.also { thumbnail ->
-            PictrsThumbnailImage(
-                thumbnail = thumbnail,
-                blur = blurNSFW && nsfwCheck(postView),
-                modifier = postLinkPicMod,
-            )
-        } ?: run {
-            Card(
-                colors = CARD_COLORS,
-                modifier = postLinkPicMod,
-                shape = MaterialTheme.shapes.large,
-            ) {
-                Box(
-                    contentAlignment = Alignment.Center,
-                    modifier = Modifier.fillMaxSize(),
+        Box {
```

Comment:
I am guessing this was needed for the clipping to work?

Reply:
Oh btw could you already add the video enum and color

Reply:
The box was added to allow the caret to overlap whatever's there -- whether it's a thumbnail or a link-placeholder.

And done!